modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/NavigationBar.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayButton from '@clayui/button';
+import ClayLayout from '@clayui/layout';
+import ClayLink from '@clayui/link';
+import ClayNavigationBar from '@clayui/navigation-bar';
+import classNames from 'classnames';
+import React from 'react';
+
+export default function NavigationBar({cssClass, inverted, navigationItems}) {
+	return (
+		<ClayLayout.ContainerFluid className={classNames('p-0', cssClass)}>
+			<ClayNavigationBar
+				inverted={inverted}
+				triggerLabel={navigationItems.find(({active}) => active)?.label}
+			>
+				{navigationItems.map(({active, href, label}, index) => {
+					const itemContent = href ? (
+						<ClayLink
+							className="nav-link"
+							displayType="unstyled"
+							href={href}
+						>
+							{label}
+						</ClayLink>
+					) : (
+						<ClayButton className="nav-link" displayType="unstyled">
+							{label}
+						</ClayButton>
+					);
+
+					return (
+						<ClayNavigationBar.Item
+							active={active}
+							data-nav-item-index={index}
+							key={label}
+						>
+							{itemContent}
+						</ClayNavigationBar.Item>
+					);
+				})}
```

Comment:
Just remember, local variables don't kill people — people with local variables kill people.

```js
				{navigationItems.map(({ active, href, label }, index) => {
					return (
						<ClayNavigationBar.Item
							active={active}
							data-nav-item-index={index}
							key={label}
						>
							{href ? (
								<ClayLink
									className="nav-link"
									displayType="unstyled"
									href={href}
								>
									{label}
								</ClayLink>
							) : (
								<ClayButton className="nav-link" displayType="unstyled">
									{label}
								</ClayButton>
							)}
						</ClayNavigationBar.Item>
					);
				})}
```

Reply:
😂 

I struggled with this... I'll inline and push 👍

Reply:
![make-it-so](https://user-images.githubusercontent.com/7074/84791939-97634300-aff3-11ea-817b-191d25fc552c.gif)

Reply:
Done so!